ctp2_code/gs/fileio/civscenarios.cpp

```diff
@@ -175,7 +175,7 @@ void CivScenarios::LoadScenarioPackData(ScenarioPack *pack, MBCHAR *packPath)
 #else
 		r = stat(scenListName, &tmpstat);
 #endif
-		if (!r) {
+		if (r == 0) {
```

Comment:
Does the new logic also fit the return value of `_stat(` from windows?

Reply:
Yes.